docroot/themes/contrib/civictheme/civictheme.provision.inc

```diff
@@ -650,8 +650,8 @@ function _civictheme_provision__blocks__mobile_navigation() {
   $theme_name = \Drupal::config('system.theme')->get('default');
 
   $block_content = civictheme_provision_create_block_content('civictheme_mobile_navigation', t('Mobile Navigation'));
-  $block_content->field_c_b_top_menu = "{$theme_name}_primarynavigation_menu";
-  $block_content->field_c_b_bottom_menu = "{$theme_name}_secondarynavigation_menu";
+  $block_content->field_c_b_top_menu = "{$theme_name}_primary_navigation";
```

Comment:
could you please advise why this was changed?

Reply:
The blocks are created as a machine id generated from label "Primary Navigation" => primary_navigation
It can verify this on current develop 
https://nginx-php.develop.civic.au2.amazee.io/

https://nimb.ws/7qmpxk